src/widgets/reasonList.test.js

```diff
@@ -80,7 +80,7 @@ const regionOneReportC = {
     'New Program Option',
     'New Staff / Turnover',
     'Ongoing Quality Improvement',
-    'Planning/Coordination (also TTA Plan Agreement)',
+    'Planning/Coordination',
```

Comment:
Should we use common values here?

Reply:
yea, guess that would have been better then repeating the definition over and over again. Should I make that change?